src/index.html

```diff
@@ -86,7 +86,7 @@ <h1>SVGOMG</h1>
       </div>
       <div class="toolbar">
         <div class="menu-toolbar-item">
-          <button class="unbutton menu-btn" type="button">
+          <button class="unbutton menu-btn" type="button" title="Menu" aria-controls="menu">
```

Comment:
`aria-controls` is mostly pointless, as it's hardly supported. https://a11ysupport.io/tech/aria/aria-controls_attribute